### PR TITLE
HHH-14660 Deprecate component mappings with different attributes for the same class

### PR DESCRIPTION
**Pull request by beikov**
https://hibernate.atlassian.net/browse/HHH-14660

### CODE REVIEW

**Review by dreab8 — APPROVED** [review]
LGTM

File: hibernate-core/src/main/java/org/hibernate/internal/EntityManagerMessageLogger.java [hunk 1]
```diff
@@ -133,4 +133,12 @@ void unableToLocateStaticMetamodelField(
 	)
 	void duplicatedPersistenceUnitName(String name);
 
+	@LogMessage(level = WARN)
```
Comment (sebersole) [inline hibernate-core/src/main/java/org/hibernate/internal/EntityManagerMessageLogger.java:136]: Please use `DeprecationLogger` for this..

**Review by sebersole — APPROVED** [review]

**Comment by sebersole** [comment]
LGTM except for the DeprecationLogger comment

File: hibernate-core/src/main/java/org/hibernate/internal/EntityManagerMessageLogger.java (hunk 1 above)
Comment (beikov) [reply hibernate-core/src/main/java/org/hibernate/internal/EntityManagerMessageLogger.java]: Fixed, thanks!

**Pull request merged**